jobs/model_create.rb

```diff
@@ -0,0 +1,53 @@
+require 'resque'
+require 'resque-status'
+require 'sinatra/activerecord'
+require 'redis'
+require_relative './reload_cache.rb'
+require_relative '../models/videos'
+require_relative '../models/articles'
+require_relative '../models/feeds'
+require_relative '../models/resources'
+
+module Jobs
+    module Models
+        class Create
+            include Resque::Plugins::Status
+            
+            #@queue = :models
```

Comment:
May want to remove obsolete code, or figure out how to assign a Resque-Status job to a queue.